### PR TITLE
Fix Issue 23194 - Add ucora to list of companies using D

### PR DESCRIPTION
**Pull request by maxhaton**

### CODE REVIEW

File: orgs-using-d.dd [hunk 1]
```diff
@@ -236,6 +236,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.tripaneer.com/news/work-with-us-software-developer-d-language-)
         )
     )
+    $(DORG ucora, www.ucora.com, ucora.svg,
```
Comment (dkorpel) [inline orgs-using-d.dd:239]: The link doesn't work, probably needs https://

File: orgs-using-d.dd [hunk 2]
```diff
@@ -236,6 +236,13 @@ $(DIVC orgs-using-d center,
...
+        We support connected, aligned, and engaged workplaces,
+        $(FA_QUOTE
```
Comment (dkorpel) [inline orgs-using-d.dd:241]: The quote doesn't show up on the website

File: orgs-using-d.dd [hunk 3]
```diff
@@ -236,6 +236,13 @@ $(DIVC orgs-using-d center,
...
+            We have been using D for 6+ years to develop and sell our GamePlanPro platform.
+            Our company has been operating for 32 years and we are committed to using D as our preferred programming language.
+        )
+    )
```
Comment (dkorpel) [inline orgs-using-d.dd:245]: There's no `$(LINK_ROW)`

**Review by dkorpel — CHANGES_REQUESTED** [review]

**Comment by dkorpel** [comment]
@mdparker I think it's up to you to approve the addition of this company.

**Comment by mdparker** [comment]
@dkorpel Once the Ddoc stuff is good to go, please do give it the greenlight.

File: orgs-using-d.dd (hunk 3 above)
Comment (maxhaton) [reply orgs-using-d.dd:245]: They don't have much to link to

**Comment by BCPaulPaul** [comment]
If it helps:

"Ucora Corporation"
"GamePlanPro(tm)"
"GamePlanPro™ is a mobile workflow platform for businesses of all sizes and types. 

Ucora is changing the way the world works."

The server platform is written entirely in D. 

https://www.ucora.com/solutions/software/
https://www.ucora.com/clients

Thank you

**Comment by maxhaton** [comment]
Awesome

**Comment by dkorpel** [comment]
Ping @maxhaton

**Comment by RazvanN7** [comment]
@maxhaton any updates on this?

**Comment by RazvanN7** [comment]
@maxhaton can you take this to the finish line?

**Comment by maxhaton** [comment]
```
make -fposix.mak html
.generated/stable_dmd-2.088.0/dmd2/linux/bin64/rdmd --compiler=.generated/stable_dmd-2.088.0/dmd2/linux/bin64/dmd -conf=.generated/stable_dmd-2.088.0/dmd2/linux/bin64/dmd.conf tools/ddoc_xml_extractor.d -i .generated/dblog_latest.xml -o .generated/dblog_latest.ddoc
No GC was initialized, please recheck the name of the selected GC ('conservative').
```
can't build the website anymore

**Comment by dkorpel** [comment]
Superseded by https://github.com/dlang/dlang.org/pull/3402

**Pull request closed without merging**